### DISCUSSION TITLE
GitHub Education approved but didn't get the benefits

### DISCUSSION THREAD

**Discussion by akshit2941** [community/community#134279] [GitHub Education]
### Hi everyone,

My GitHub education application was approved but it's been 3 days and I still haven't received any mail regarding the benefits I'll be getting after the approval.

**Comment by davevad93** [comment]
Hi @akshit2941 ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @akshit2941 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by akshit2941** [reply]
Ok 👍

**Reply by ravilovrm** [reply]
help me please
https://github.com/orgs/community/discussions/134402

**Reply by davevad93** [reply]
Hey @TUTBOROMA , I'm sorry but I can't help you beyond giving advice as I'm not from GitHub nor do I have any affiliation with the staff. I know you've been having this problem for a while since I remember a discussion of yours from last week and you've already checked the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352). Try opening a discussion explaining your issue and attaching screenshots, also try mentioning [samus-aran](https://github.com/samus-aran) who I believe is in charge of GitHub Education related stuff here in the Community.

**Comment by akshit2941** [comment]
Thanks GitHub for the education pack!
